test/pkg.jl

```diff
@@ -533,4 +533,19 @@ temp_pkg_dir() do
         @test ret === nothing && out == ""
         @test contains(err, nothingtodomsg)
     end
+
```

Comment:
I noticed (too late) that the test case above that is testing partial updates uses a similar method. I will try to include a test for this behavior there.